MAUI/Scheduler/appointments.md

```diff
@@ -1122,6 +1124,77 @@ this.Scheduler.SelectedAppointmentBackground = Brush.Orange;
 
 ![appointment-selection-background-in-maui-scheduler](images/appointments/appointment-selection-background-in-maui-scheduler.png)
 
+## Customize Appointment Border Style
+
+The appointment border style can be customized using the `AppointmentBorderStyle` property, which allows you to modify the `Stroke`, `CornerRadius`, and `StrokeThickness` of appointments in the Scheduler.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/weekview.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="8 14" %}
+
+var appointments = new ObservableCollection<SchedulerAppointment>();
+appointments.Add(new SchedulerAppointment()
+{
+    Subject = "meeting",
+    StartTime = DateTime.Now,
+    EndTime = DateTime.Now.AddHours(1),
+});
+
+this.Scheduler.AppointmentsSource = appointments;
+var appointmentBorderStyle = new SchedulerAppointmentBorderStyle ()
+{
+    Stroke = Colors.Red,
+    CornerRadius = 5,
```

Comment:
Add new property in mapping table
https://help.syncfusion.com/maui/scheduler/appointments#scheduler-appointment-source-mapping